.eslintrc.js

```diff
@@ -61,6 +61,12 @@ module.exports = {
         'testing-library/render-result-naming-convention': 'warn',
       },
     },
+    {
+      files: ['./packages/**/*.{js,jsx,ts,tsx}'],
+      rules: {
+        'no-console': ['error', { allow: ['warn', 'error'] }],
```

Comment:
Shipped code can't have any console method except console.warn and console.error.

Reply:
Yeah, I like that. @zarahzachz, what do you think?

Reply:
This seems like a good enhancement